### ISSUE TITLE
Accessibility: Color Blind friendly theme

### ISSUE THREAD

**Issue by davideghz** [a85/POSTMan-Chrome-Extension#4455]
Hi, not sure whether this is the right place to post about this or not, but it would be great to have the possibility to chose which color to use for http verbs in the left sidebar:

To me, the GET & POST labels look like they have exactly the same color and are not easily recognizable at first glance :)
![image](https://user-images.githubusercontent.com/6973157/38767655-a86f79aa-3fe5-11e8-9782-0a1e11f83ea7.png)

**Comment by sdnts** [comment]
Hi @davideghz! This is indeed the right place to post feature requests :)
We're working on customizable themes currently, so when that goes out, you should be able to make one yourself! I'll mark this as a feature request and update this thread when we have more info!

**Comment by sigwinch28** [comment]
Any update on this? I especially struggle to tell the difference between any reds and greens used in the app.
